build-system/pr-check.js

```diff
@@ -207,6 +207,12 @@ function main(argv) {
     return 0;
   }
 
+  if (!(files.some(x => x === 'package.json') &&
```

Comment:
The logic here fails if PR does not include either of them. We want logical XOR here. Maybe do:

if( files.includes('package.json') ? files.includes('yarn.lock') : !files.includes('yarn.lock') )